mailpoet/assets/js/src/automation/editor/components/panel/activate-panel.tsx

```diff
@@ -0,0 +1,99 @@
+import { useDispatch, useSelect } from '@wordpress/data';
+import { Button } from '@wordpress/components';
+import { closeSmall } from '@wordpress/icons';
+import { __ } from '@wordpress/i18n';
+import { storeName } from '../../store';
+import { WorkflowStatus } from '../../../listing/workflow';
+import { MailPoet } from '../../../../mailpoet';
+
+function PreStep({ onClose }): JSX.Element {
+  const { activate } = useDispatch(storeName);
+  return (
+    <>
+      <div className="mailpoet-automation-activate-panel__header">
+        <div className="mailpoet-automation-activate-panel__header-activate-button">
+          <Button variant="primary" onClick={activate}>
+            {__('Activate', 'mailpoet')}
+          </Button>
+        </div>
+
+        <div className="mailpoet-automation-activate-panel__header-cancel-button">
+          <Button variant="secondary" onClick={onClose}>
+            {__('Cancel', 'mailpoet')}
+          </Button>
+        </div>
+      </div>
+
+      <div className="mailpoet-automation-activate-panel__body">
+        <p>
+          <strong>{__('Are you ready to activate?', 'mailpoet')}</strong>
+        </p>
+        <p>{__('Double-check your settings before activating', 'mailpoet')}</p>
```

Comment:
I think we can add a `.` at the end of the sentence since WP seems to do so for posts:

> Double-check your settings before publishing.